Assets/Scripts/Model/Choices/Steal/CannotSteal.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace model.choices.steal
+{
+    public class CannotSteal : IStealOption
+    {
+        Task IStealOption.Perform(Game game) => Task.CompletedTask;
```

Comment:
`Task` is just a `void` that works with `async`.

Reply:
Makes sense, thanks